Automattic/wp-calypso
# Jetpack Onboarding: Update illustrations in JPO flow

This PR updates three illustration assets in three of the steps of the Jetpack Onboarding (JPO), updates the filenames and paths in the respective steps jsx files, and tweaks the styling a little bit. The `jetpack-start.svg` (formerly known as `site-title.svg`) will also be used in our updated Plans view (cc @AnnaMag who is working on that)

JPO Site title step (uses `jetpack-start.svg`)
<img width="556" alt="screen shot 2018-03-02 at 12 14 55 pm" src="https://user-images.githubusercontent.com/204742/36917529-49625402-1e14-11e8-9189-a7dac3ba6821.png">

JPO Business address step (uses `jetpack-business-address.svg`)
<img width="590" alt="screen shot 2018-03-02 at 12 15 01 pm" src="https://user-images.githubusercontent.com/204742/36917534-4c9e8622-1e14-11e8-8ca7-d159b3f26065.png">

JPO Stats step (uses `jetpack-stats.svg`)
<img width="598" alt="screen shot 2018-03-02 at 12 15 06 pm" src="https://user-images.githubusercontent.com/204742/36917540-5167d55a-1e14-11e8-937e-4240e6b331a9.png">

**Testing instructions:**
* Checkout this branch
* Pick a Jetpack site.
* Head to /wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development on the Jetpack site to start the onboarding flow.
* On the site title step, verify the illustration looks the same as the above "After" shot.
* In the site type step, select "Business".
* Skip to the business address step and verify the illustration looks the same as the above "After" shot.
* Skip to the stats step and verify the illustration looks the same as the above "After" shot.
